tests/parse.lua

```diff
@@ -76,19 +79,19 @@ describe('parse', function()
     end)
 
     test('directives', function()
-      local operation = parse('query{}').definitions[1]
+      operation = parse('query{}').definitions[1]
```

Comment:
To be honest, I use redefinitions in my code and always disable this luacheck warning. It sometimes helps to keep code blocks being uniformly formatted.